### PR TITLE
Dp2 1/user roles

### PR DESCRIPTION
**Pull request by grinderek**
Acceptance criteria:
Setup the following URLs for admin and user: 
URL for admin:
[website_home_page]/admin - admin's homepage 
[website_home_page]/admin/[page_name] - other pages
URL for user:
[website_home_page]- user's homepage 
[website_home_page]/[page_name] - other pages

### CODE REVIEW

File: .idea/DP.iml [hunk 1]
```diff
@@ -40,6 +40,7 @@
     <orderEntry type="library" scope="PROVIDED" name="activestorage (v6.1.4.4, rbenv: 3.0.2) [gem]" level="application" />
```
Comment (DmitryGolubko) [inline .idea/DP.iml]: Add .idea folder to .gitignore and remove this folder from gitHub repo. https://stackoverflow.com/questions/7927230/remove-directory-from-remote-repository-after-adding-them-to-gitignore

File: config/environments/development.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
-require "active_support/core_ext/integer/time"
+# frozen_string_literal: true
```
Comment (DmitryGolubko) [inline config/environments/development.rb:1]: What do frozen_string_literal stand for? No need to remove it, just a question.

**Review by DmitryGolubko — CHANGES_REQUESTED** [review]

File: config/environments/development.rb (hunk 2 above)
Comment (grinderek) [reply config/environments/development.rb:1]: It is not my wish, Rubocop did it) So it makes all strings frozen - I can't call methods on strings which modifes them. I don't know why it's necessary, there must be something in Ruby Style Guides

Comment (DmitryGolubko) [reply config/environments/development.rb:1]: https://stackoverflow.com/questions/37799296/ruby-what-does-the-comment-frozen-string-literal-true-do

**Review by DmitryGolubko — APPROVED** [review]

**Pull request merged**